internal/slack/formatter/formatter.go

```diff
@@ -113,9 +113,12 @@ func FormatMessage(text string, options FormatOptions) []slack.MsgOption {
 				case "divider":
 					slackBlock = slack.NewDividerBlock()
 				case "context":
-					var context slack.ContextBlock
-					if err := json.Unmarshal(blockJSON, &context); err == nil {
-						slackBlock = context
+					var contextBlock struct {
```

Comment:
Nice fix! The previous code tried to unmarshal directly into `slack.ContextBlock` which doesn't work properly. Extracting `Elements` and reconstructing with `NewContextBlock` is the right approach.